CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/IncrementSlider` `sliderRef` prop to pass reference to the slider node
 - `agate/Slider` `sliderRef` prop to pass reference to the slider node
 
+### Removed
+
+- `agate/ProgressBarTooltip` unused code block referring to a third orientation option
+
```

Comment:
```suggestion
```
It seems to me that it would be better to remove this changelog. This code change is not a case of reducing the orientations of the progressBar supported by Agate, but originally two orientations were supported. And this is a refactoring to modify the code accordingly. So I think we don't have to add a changelog of this fix.